daemon/api_snaps.go

```diff
@@ -294,6 +294,10 @@ func snapInstall(inst *snapInstruction, st *state.State) (string, []*state.TaskS
 		return "", nil, err
 	}
 
+	if inst.IgnoreValidation {
+		flags.IgnoreValidation = true
+	}
+
```

Comment:
for consistency this should move to installFlags, no?

Reply:
Good point, done.